style.css

```diff
@@ -1,3 +1,21 @@
+body {
+font-family: 'Roboto', sans-serif;
+display: flex;
+flex-direction: column;
+background-color: #d1ecff;
+align-items: center;
+}
+#welcome {
+    width: 80%;
+    max-width: 900px;
```

Comment:
px is still pretty common for page widths and general layout afaik, mobile devices commonly scale these values with a pixel ratio based on pixel density on the screen with a reasonably sane default.

For example the Samsung Galaxy S4 has a full-HD resolution screen, and uses a CSS pixel ratio of 3 - making the effective size of that screen 360x640.

The defaults can be overriden, however they seem reasonable:

![image](https://user-images.githubusercontent.com/1552366/27954722-7fef5808-6311-11e7-9994-84f3f6cf3a12.png)